### PR TITLE
Clarify Connector implementation links

### PR DESCRIPTION
**Pull request by sappenin**
@LoisRP

### CODE REVIEW

File: _content/setup-wallets.md [hunk 1]
```diff
@@ -11,10 +11,14 @@ This tutorial describes two ways to create an Interledger account to send and re
...
 
-If you want to send actual value in your app with an Interledger account, refer to the [Rust implementation of Interledger](http://interledger.rs), or the Java implementation of Interledger at [Hyperledger Quilt](https://www.hyperledger.org/projects/quilt). Because these efforts are open source, your contributions are welcome.
-
 You can also perform these same tasks programmatically. See [Set up Interledger accounts programmaticallly](setup-wallets-programmatically.html).
 
+If you want to send actual value in your app with an Interledger account, refer to the following open-source projects (contributions welcome):
+
+* [Java Connector](https://connector.interledger4j.dev)  — Actively maintained by Xpring and recommended for new production deployments.
+* [Rust Connector](http://interledger.rs)  — Not actively maintained, but recommended to get started for its ease of use and feature-set.
+* [Javascript Connector](https://github.com/interledgerjs/ilp-connector)  — Not actively maintained, byt battle-tested in production.
```
Comment (kincaidoneil) [inline _content/setup-wallets.md]:
```suggestion
* [Javascript Connector](https://github.com/interledgerjs/ilp-connector)  — Not actively maintained, but battle-tested in production.
```

Comment (sappenin) [reply _content/setup-wallets.md]: Fixed!

File: _content/setup-wallets.md [hunk 2]
```diff
@@ -11,10 +11,14 @@ This tutorial describes two ways to create an Interledger account to send and re
...
 
-If you want to send actual value in your app with an Interledger account, refer to the [Rust implementation of Interledger](http://interledger.rs), or the Java implementation of Interledger at [Hyperledger Quilt](https://www.hyperledger.org/projects/quilt). Because these efforts are open source, your contributions are welcome.
-
 You can also perform these same tasks programmatically. See [Set up Interledger accounts programmaticallly](setup-wallets-programmatically.html).
 
+If you want to send actual value in your app with an Interledger account, refer to the following open-source projects (contributions welcome):
+
+* [Java Connector](https://connector.interledger4j.dev)  — Actively maintained by Xpring and recommended for new production deployments.
+* [Rust Connector](http://interledger.rs)  — Not actively maintained, but recommended to get started for its ease of use and feature-set.
+* [Javascript Connector](https://github.com/interledgerjs/ilp-connector)  — Not actively maintained, but battle-tested in production.
```
Comment (matdehaast) [inline _content/setup-wallets.md]: Can we remove the `Not actively maintained`. Currently all production ILP nodes are running this and some new partners are about to deploy this.

File: setup-wallets.html [hunk 3]
```diff
@@ -113,8 +113,13 @@ <h1 id="main-page-header">Get Started With Interledger Accounts</h1>
...
 <p>You can also perform these same tasks programmatically. See <a href="setup-wallets-programmatically.html">Set up Interledger accounts programmaticallly</a>.</p>
+<p>If you want to send actual value in your app with an Interledger account, refer to the following open-source projects (contributions welcome):</p>
+<ul>
+<li><a class="external-link" href="https://connector.interledger4j.dev" target="_blank">Java Connector <i aria-hidden="true" class="fa fa-external-link"></i></a>  — Actively maintained by Xpring and recommended for new production deployments.</li>
+<li><a class="external-link" href="http://interledger.rs" target="_blank">Rust Connector <i aria-hidden="true" class="fa fa-external-link"></i></a>  — Not actively maintained, but recommended to get started for its ease of use and feature-set.</li>
+<li><a class="external-link" href="https://github.com/interledgerjs/ilp-connector" target="_blank">Javascript Connector <i aria-hidden="true" class="fa fa-external-link"></i></a>  — Not actively maintained, byt battle-tested in production.</li>
```
Comment (matdehaast) [inline setup-wallets.html]: Same as above.

**Pull request merged**